### ISSUE TITLE
Question regarding data used for training and availiable on zenodo

### ISSUE THREAD

**Issue by yogeshverma1998** [yehlincho/absolute-stability-predictor#4]
Hi,

Thanks for sharing this great work and the dataset. I have a couple of questions about the data available on Zenodo, specifically how the files differ and which one was used for fine-tuning:

1. In the Zenodo dataset (https://zenodo.org/records/19411306), was `dmsv4_filtered_train_splits.csv` the file used for fine-tuning, since it includes a `deltaG` column? Or was `230515_K50dG_dmsv4_dmsv5_dmsv7_concat260429.csv` used instead, which also has a `deltaG` column? The two files differ in the number of entries, so I wanted to confirm which was actually used, and which column.

2. Also, did you use the Sigmoid-Corrected Stability Head only for the ΔΔG ones, or is it used throughout the paper for the results shown?

Thanks in advance for your time!

**Comment by grocklin** [comment]
All training was done using just dmsv4_filtered_train_splits.csv, but I think all the data in the larger file could be used with some basic quality filtering. All models were trained with the sigmoid correction, but for Fig 4-6 the sigmoid transformation was removed, I believe @yehlincho .
